lib/src/showcase.dart

```diff
@@ -188,7 +188,26 @@ class _ShowcaseState extends State<Showcase> with TickerProviderStateMixin {
     });
 
     if (activeStep == widget.key) {
-      _slideAnimationController.forward();
+      if (_slideAnimationController == null) {
```

Comment:
use null aware operator instead of null check

Reply:
Added null aware operator